sofagym/envs/Maze/tripod.py

```diff
@@ -0,0 +1,167 @@
+                      translation=[0.0, 30, 0.0], rotation=[90, 0, 0], collisionGroup=1)
+
+    self.addCollision = addCollision
+
+    return self
+
+
+def createScene(rootNode):
+    scene = Scene(rootNode, gravity=[0.0, -9810, 0.0], iterative=False)
+    scene.addMainHeader()
+    scene.addObject('DefaultVisualManagerLoop')
+    scene.addObject('FreeMotionAnimationLoop')
+    scene.addObject('GenericConstraintSolver', maxIterations=50, tolerance=1e-5)
+    scene.Simulation.addObject('GenericConstraintCorrection')
+    scene.Settings.mouseButton.stiffness = 10
+    scene.Simulation.TimeIntegrationSchema.rayleighStiffness = 0.05
+    scene.VisualStyle.displayFlags = "showBehavior"
+    scene.dt = 0.01
+
+    # Add the tripod
+    tripod = scene.Modelling.addChild(Tripod())
+    tripod.BoxROI0.drawBoxes = True
+    tripod.BoxROI1.drawBoxes = True
+    tripod.BoxROI2.drawBoxes = True
+
+    # Use this to activate some rendering on the rigidified object ######################################
+    setData(tripod.RigidifiedStructure.RigidParts.dofs, showObject=True, showObjectScale=10, drawMode=2)
+    setData(tripod.RigidifiedStructure.RigidParts.RigidifiedParticules.dofs, showObject=True, showObjectScale=1,
+            drawMode=1, showColor=[1., 1., 0., 1.])
+    setData(tripod.RigidifiedStructure.DeformableParts.dofs, showObject=True, showObjectScale=1, drawMode=2)
+    #####################################################################################################
+
+    scene.Simulation.addChild(tripod)
+
+    def myanimate(targets, factor):
+        for arm in targets:
+            arm.angleIn = -factor * math.pi / 4.
+
+    animate(myanimate, {"targets": [tripod.ActuatedArm0, tripod.ActuatedArm1, tripod.ActuatedArm2]}, duration=1)
+
+    # Temporary additions to have the system correctly built in SOFA
+    # Will no longer be required in SOFA v23.12
+    scene.Simulation.addObject('MechanicalMatrixMapper',
+                               name="deformableAndFreeCenterCoupling",
+                               template='Vec3,Rigid3',
+                               object1=tripod["RigidifiedStructure.DeformableParts.dofs"].getLinkPath(),
+                               object2=tripod["RigidifiedStructure.FreeCenter.dofs"].getLinkPath(),
+                               nodeToParse=tripod["RigidifiedStructure.DeformableParts.MechanicalModel"].getLinkPath())
+
+    # Temporary additions to have the system correctly built in SOFA
+    # Will no longer be required in SOFA v23.12
+    for i in range(3):
+        scene.Simulation.addObject('MechanicalMatrixMapper',
+                                   name="deformableAndArm{i}Coupling".format(i=i),
+                                   template='Vec1,Vec3',
+                                   object1=tripod[
+                                       "ActuatedArm" + str(i) + ".ServoMotor.Articulation.dofs"].getLinkPath(),
```

Comment:
`MechanicalMatrixMapper` is no longer needed. You can remove all references. Note that this is not true for v22.12 and earlier